flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvFormatFactoryTest.java

```diff
@@ -206,6 +206,29 @@ public void testDeserializeWithEscapedFieldDelimiter() throws IOException {
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testDeserializeWithDisableQuoteCharacter() throws IOException {
```

Comment:
`CsvFormatFactoryTest` is used to verify the received options can be converted into expected serializer/deserializer. The parse/unparse logic should be tested in `CsvRowDataSerDeSchemaTest`. 

Could you add a test in `CsvRowDataSerDeSchemaTest` to test deserialization with quote disabled ? And add a DeserializationSchema assertEquals test in `CsvFormatFactoryTest#testDisableQuoteCharacter`?

Reply:
ok

Reply:
We can remove this method now, because it has been covered by the new introduced other 2 tests.